ERCS/erc-7902.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7902
+title: Wallet Capabilities for Account Abstraction
+description: New Wallet Capabilities providing a way for dApps and Account Abstraction wallets to communicate
```

Comment:
> [...] communicate

This is a bit vague. I know there isn't a lot of space in the description, but I'd like you to try to pack a bit more detail about what is communicated between the dapp and wallet in here.

Reply:
Tried to rephrase to make the description more informative